devicetypes/smartthings/fidure-thermostat.src/fidure-thermostat.groovy

```diff
@@ -12,6 +12,11 @@ metadata {
 		capability "Actuator"
 		capability "Temperature Measurement"
 		capability "Thermostat"
+		capability "Thermostat Heating Setpoint"
+		capability "Thermostat Cooling Setpoint"
+		capability "Thermostat Operating State"
```

Comment:
This DTH has a custom attribute called `thermostatOperatingState` that I think should be removed. Also the values it uses in `thermostatOperatingState` events don't match the capability exactly and should be updated.